packages/bundlers/experimental/src/ExperimentalBundler.js

```diff
@@ -1002,29 +1002,36 @@ function createIdealGraph(
     }
   }
 
+  // Build a map of bundleGroups to bundles in the group -- this includes all referenced bundles.
+  let bundleGroups = bundleGraph.getNodeIdsConnectedFrom(rootNodeId);
+  let bundleGroupToBundles = new Map<NodeId, Set<NodeId>>();
+  for (let bundleGroupId of bundleGroups) {
+    let bundleIds = new Set();
+    bundleGraph.traverse(nodeId => {
+      bundleIds.add(nodeId);
+    }, bundleGroupId);
+    bundleGroupToBundles.set(bundleGroupId, bundleIds);
+  }
```

Comment:
Shouldn't all bundles in a bundle group be directly attached to the entry bundle node? When is that not the case?

Reply:
It doesn't look like they're connected at this stage yet -- the edges are created in this step of `decorateLegacyGraph`:
https://github.com/parcel-bundler/parcel/blob/v2/packages/bundlers/experimental/src/ExperimentalBundler.js#L253-L276